addon/components/stacked-vertical-bar-chart.js

```diff
@@ -37,11 +45,12 @@ const StackedVerticalBarChartComponent = ChartComponent.extend(LegendMixin,
     // to do is have the equation for bar width based on padding and use that
     // to set the padding exactly.
     var scale = d3.scale.linear().domain([1, 8]).range([1.25, 0.25]).clamp(true);
-    return scale(this.get('numBars'));
+    return scale(this.get('numSlices'));
   }),
 
-  numBars: Ember.computed('xBetweenGroupDomain', 'xWithinGroupDomain', function() {
-    return this.get('xBetweenGroupDomain.length') * this.get('xWithinGroupDomain.length') || 0;
+  numSlices: Ember.computed('barNames.length', 'allSliceLabels.length',
```

Comment:
Can you add documentation for this? I had hard time trying to understand why this is equal to the product of `barNames` length and `allSliceLabels`.